apps/rush-lib/src/logic/deploy/DeployManager.ts

```diff
@@ -593,6 +599,18 @@ export class DeployManager {
 
       await this._makeBinLinksAsync(deployState);
     }
+    if (deployState.scenarioConfiguration.json.folderNameToCopy !== undefined) {
+      const sourceFolderPath: string = path.resolve(
+        this._rushConfiguration.rushJsonFolder,
+        deployState.scenarioConfiguration.json.folderNameToCopy
```

Comment:
The docs should mention that this path is resolved relative to the repository root folder.  I'd also suggest to call it `folderToCopy` since it is generally a path (and not just the folder's name).